src/Microsoft.Health.Fhir.SqlServer/Features/Storage/JobInfoExtensions.cs

```diff
@@ -39,7 +39,7 @@ public static async Task<IReadOnlyList<JobInfo>> ReadJobInfosAsync(this SqlDataR
             return outcome;
         }
 
-        private static JobInfo LoadJobInfo(SqlDataReader sqlDataReader)
+        public static JobInfo LoadJobInfo(SqlDataReader sqlDataReader)
```

Comment:
I believe that, as we are making this method public, we should validate if 'sqlDataReader' is not null.

Reply:
Done